src/pages/history/history.jsx

```diff
@@ -0,0 +1,23 @@
+import { VideoCard } from "components"
+import { useHistory, useVideo } from "contexts"
+import { useEffect } from "react"
+import "./history.css"
+
+const History = () => {
+    const {state} = useVideo()
+    const {getHistory,clearHistory} =useHistory()
+    useEffect(()=>{
+        getHistory()
+    })
```

Comment:
try to give one dependency array in useEffect hooks

Reply:
Thanks, I missed it.